xmake/modules/package/manager/cmake/configurations.lua

```diff
@@ -23,6 +23,7 @@ function main()
     return
     {
         link_libraries = {description = "Set the cmake package dependencies, e.g. {\"abc::lib1\", \"abc::lib2\"}"},
+        include_dirs   = {description = "Set the cmake package include directories, e.g. {\"${ZLIB_INCLUDE_DIRS}\"}"},
```

Comment:
please use `include_directories` Consistent with the cmake interface.

Reply:
I tried but the variable is too long and will break the alignment, change all?

Reply:
change all